Add depth option for OSTree repositories (#3343)
theforeman/foreman-documentation: guides/common/modules/proc_importing-ostree-content.adoc

The excludes are evaluated after the includes.
. Optional: In the *Include Refs* field, enter a list of OSTree head refs separated with commas to include in importing to {Project}.
For example `fedora/x86_64/coreos/stable`.

> . Optional: Add a number in the *Depth* field, this is an option to specify how many commits to traverse.

Reviewer: ```suggestion
. Optional: In the *Depth* field, enter the number of commits to traverse.
```

just a suggestion.

Author: Yup I agree that's a better wording 👍🏻
